src/main/java/org/terasology/logic/inventory/ItemCommands.java

```diff
@@ -96,16 +114,10 @@ public String give(
             }
 
         } else if (matches.size() > 1) {
-            StringBuilder builder = new StringBuilder();
-            builder.append("Requested item \"");
-            builder.append(itemPrefabName);
-            builder.append("\": matches ");
-            Joiner.on(" and ").appendTo(builder, matches);
-            builder.append(". Please fully specify one.");
-            return builder.toString();
+            return buildAmbiguousobjectIdString("item", itemPrefabName, matches);
```

Comment:
Type in camel-case wording:
```suggestion
            return buildAmbiguousObjectIdString("item", itemPrefabName, matches);
```